esx_service/vmci/vmci_client.c

```diff
@@ -51,6 +52,8 @@ typedef struct be_request {
 } be_request;
 
 #define MAXBUF 1024 * 1024 // Safety limit. We do not expect json string > 1M
+#define MAX_CLIENT_PORT 1023 // Last privileged port
+#define START_CLIENT_PORT 1000 // Where to start client port
```

Comment:
out of curiosity - why 1000 ? 
A bigger question is "would the range of 23 be enough for the timing to be right" ?